docs/usage/config-overview.md

```diff
@@ -1,12 +1,18 @@
 # Renovate configuration overview
 
-When Renovate runs on a repository, the final config used is derived from the:
-
-- Default config
-- Global config
-- Inherited config
-- Repository config
-- Resolved presets referenced in config
+When Renovate runs on a repository, the final config used is derived from the following config locations. The below is the order of precedence:
+
+1. Default config
+2. Global config
+   2.1 File config
+   2.2 Environment config
+   2.3 CLI config
```

Comment:
I still see this "rich" diff: 
![image](https://github.com/user-attachments/assets/f0328fc8-a93d-4e81-8cf3-4a46846e3fd8)

Is the diff wrong?

Reply:
I don't know how this looks in the published docs... 😞

The published docs use a Python Markdown variant. But the rich diff on GitHub assumes the input is GitHub Flavored Markdown.

I remember us having problems with list formatting as Python Markdown uses 4 spaces. But GitHub-flavored markdown uses different conventions for some list types.

Reply:
> Maybe should we use a) b) c) instead?

Sounds good!

It will probably not fix the problem of the layout, but it's easier to read and remember.

A number and a letter seems better than a number and then a _slightly_ different number.